### PR TITLE
fix(rpc)!: various rpc breaks

### PR DESCRIPTION
**Pull request by distractedm1nd**
RPC Break feature branch. Needs to not be squashed

### CODE REVIEW

**Comment by distractedm1nd** [comment]
idk why it added me to every commit, can I change that? It could be from the rebase?

**Review by vgonkivs — CHANGES_REQUESTED** [review]
Did you rebase it on the latest main? I feel there should be issues inside `nodebuilder/das/cmd`. Also, could you please go through all cmds and ensure that nothing were broken?

File: nodebuilder/node/admin.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/celestiaorg/celestia-node/libs/authtoken"
 )
 
-const APIVersion = "v0.2.2"
+const APIVersion = "v0.11.0"
```
Comment (Wondertan) [inline nodebuilder/node/admin.go:13]: So now we have to PR a string change every time we update the version. I wonder if there is a way to automate that.

**Review by Wondertan — DISMISSED** [review]

**Comment by vgonkivs** [comment]
cc @jcstein for visibility
Recent changes in the node's API:
* ProbalitiyOfAvailability endpoint(and cmd) was completely removed;
* GetShare, GetEDS, GetSharesByNamespace, and SharesAvailable now take not the DAH, but the full ExtendedHeader
* fraud.Subscribe returns `<-*Proof `instead of `<-Proof`

File: nodebuilder/node/admin.go (hunk 1 above)
Comment (distractedm1nd) [reply nodebuilder/node/admin.go:13]: Dont think so unfortunately :/ This is one unforeseen issue with #2768

**Review by renaynay — DISMISSED** [review]

**Review by Wondertan — DISMISSED** [review]

**Review by ramin — DISMISSED** [review]

**Review by walldiss — APPROVED** [review]

**Review by vgonkivs — APPROVED** [review]

**Review by renaynay — DISMISSED** [review]

**Review by Wondertan — APPROVED** [review]

**Review by ramin — DISMISSED** [review]

**Review by vgonkivs — DISMISSED** [review]

**Review by renaynay — APPROVED** [review]

**Review by Wondertan — APPROVED** [review]

**Pull request merged**